docs/portal_config.md

```diff
@@ -714,6 +714,7 @@ Below is an example, with inline comments describing what each JSON block config
   "studyViewerConfig": [],//See docs/study_viewer.md for more details.
   "useArboristUI": false, // optional; set true to enable Arborist UI; defaults to false if absent
   "hideSubmissionIfIneligible": true, // optional; only works if Arborist UI is enabled; if set to true, link/buttons to /submission page will be hidden to users who don't have permissions to submit data; defaults to false if absent
+  "profilePageTitle": "", // title to display at top of Profile Page
```

Comment:
is this a 508 requirement too? If so maybe we should make it to have a valid default value

Reply:
yes the requirement is to have a level 1 heading on all pages, normally meaning the page should have a H1 that lets the user know where they are. I was worried by having these by default might upset stakeholders. If you are fine with it i can make a default for this as well as the Workspace page

Reply:
yeah if that is a 508 req i think we should definitly have a default value for it
and we can pitch to our stakeholders about the change

Reply:
made the updates, shows by default now